### DISCUSSION TITLE
Academic Research - Green Software Development/Operation

### DISCUSSION THREAD

**Discussion by NigelM-T** [Green-Software-Foundation/discussions#114] [Ask Anything]
Hi Green Software Foundation community, Happy New Year!

I'm currently working on an academic research project which hopes to examine:

- How organisations are managing their environmental responsibilities during software development and operation.
- What are the main reported obstacles for Green Software development/operation being taken more seriously.
- How software professionals are practically contributing to this area.

I'm conducting qualitative research through interviews in January/February, and hoping to engage with some participants who are experienced in this area, and others who are less experienced. Participation and responses will be anonymised and represented in the research with a numerical identifier, and consent can be withdrawn at any time.

I wondered if anyone at GSF would be willing to volunteer to be interviewed on this topic. I expect interviews to last around 45mins, and I'm looking for around 1-5 participants experienced in this area. If anyone is interested in being interviewed please let me know - or if you have any questions on this, please let me know also.

I hope this research will be a contribution to this space, and a great opportunity for me to learn more about it.

Many thanks,
Nigel Matthews

**Comment by russelltrow** [comment]
Hi @NigelM-T it sounds like you are working on an incredibly interesting and meaningful projects, thanks so much for sharing with us.

@NAMRATA-WOKE how could we best share this with our member? Something for the next member newsletter?
